Scripts/RemediationScripts/Remediate-Azure_AppService_DP_Use_Secure_FTP_Deployment.ps1

```diff
@@ -0,0 +1,1132 @@
+<###
```

Comment:
![image](https://user-images.githubusercontent.com/52916530/190473931-f585c033-0798-4081-baa5-ea588b9db293.png)

Reply:
Changed the index name as per above suggestion.